python/lsst/pipe/analysis/analysis.py

```diff
@@ -1094,6 +1098,91 @@ def plotQuiver(self, catalog, description, plotInfoDict, areaDict, log, cmap=plt
 
         yield Struct(fig=fig, description=description, stats=stats, statsHigh=None, dpi=150, style="quiver")
 
+    def plotRhoStatistics(self, description, plotInfoDict, log, treecorrParams,
+                          stats, butler=None, camera=None,
+                          ccdList=None, tractInfo=None, patchList=None, hscRun=None, matchRadius=None,
+                          zpLabel=None, forcedStr=None, postFix="", flagsCat=None, uberCalLabel=None):
+        """ Plot Rho Statistics """
+
```

Comment:
Please could you add proper docs here.

Reply:
Since other similar functions had one-line doc strings, I left it at that. The function signature used here is based on other similar functions, and the arguments are not explained in any of the doc strings. So I don't know enough about them to include it here. I could include a partially detailed doc string, but perhaps best when revamping more of pipe_analysis in a dedicated ticket?